app/scripts/lib/notification-manager.js

```diff
@@ -12,7 +18,9 @@ export default class NotificationManager {
    */
 
   constructor() {
+    super();
     this.platform = new ExtensionPlatform();
+    extension.windows.onRemoved.addListener(this._onWindowClosed.bind(this));
```

Comment:
Nit: We've been using the `ExtensionPlatform` module as a way to encapsulate our extension API interactions, like this one. In keeping with that, this might be better off as an `ExtensionPlatform` method. Something like `addOnRemovedListener`.

Not a blocker, as we aren't even consistently doing that yet today (we use `extensionizer` directly in a few places). But since `this.platform` is already here it might be easy enough.